projects/storefrontlib/src/lib/cms-lib/navigation/navigation-ui.component.scss

```diff
@@ -28,7 +28,7 @@ $cx-navigation-text-case: capitalize !default;
 $cx-navigation-item-trbl-padding: 0.5rem 0 !default;
 $cx-navigation-item-border: 1px solid !default;
 $cx-navigation-link-trbl-padding: 0.2rem 0 !default;
```

Comment:
I was also thinking adding a padding to the left and right so when you make the page narrower, the navigation elements don't get right one after the other before wrapping.

Reply:
Currently it wraps at 
![screen shot 2019-01-15 at 2 43 42 pm](https://user-images.githubusercontent.com/10605724/51205843-ce03ce00-18d4-11e9-896b-54eee5f55f16.png)

I would like to wrap at 
![screen shot 2019-01-15 at 2 47 26 pm](https://user-images.githubusercontent.com/10605724/51205888-e7a51580-18d4-11e9-9627-26e032b9dcff.png)